frontend/src/main/java/com/google/tripmeout/frontend/servlet/TripServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.tripmeout.frontend.servlet;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.TripModel;
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import com.google.tripmeout.frontend.error.TripNotFoundException;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Singleton
+public final class TripServlet extends HttpServlet {
+  // The Trip get and delete servlet
+  private static final long serialVersionUID = 1L;
```

Comment:
Nit: keep `static` (non-member) fields together non-`static` (member) fields together and separate the two groups with whitespace. Likewise, keep `final` and non-`final` fields grouped and separate.

This part is more about taste, but: declare/define `static` fields before non-`static` fields and `final` fields before non-`final` fields.

For example:
```java
private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";

private final InMemoryTripModelStorage storage;
private final Gson gson;
```